test/helpers.ts

```diff
@@ -1,24 +1,37 @@
 import * as rpc from "vscode-jsonrpc"
 import { TextEditor } from "atom"
+import AutoLanguageClient from "../lib/auto-languageclient"
+import { LanguageClientConnection, Disposable, Event } from "../lib/languageclient"
+import { LanguageServerProcess } from "../lib/server-manager"
+import { spawn } from "spawk"
```

Comment:
I don't think we actually need spawk with jasmine.

```js
spyOn(require("child_process"), "spawn").and.callThrough()
```

That should be all we need.

Reply:
But `spawn` will fail with this. We are not spawning a real language process.

Reply:
Ya I guess spawk handles a lot of the mocking process.